azkaban-common/src/main/java/azkaban/executor/IntHandler.java

```diff
@@ -0,0 +1,24 @@
+package azkaban.executor;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import org.apache.commons.dbutils.ResultSetHandler;
```

Comment:
Do we need the changes in this file? FETCH_EXECUTION_LOGS_COUNT is not used in this change.

Reply:
My bad. I am removing this change.